src/main/java/uk/gov/justice/hmpps/prison/security/AuthenticationFacade.java

```diff
@@ -55,8 +56,7 @@ public static boolean hasRoles(final String... allowedRoles) {
     }
 
     public boolean isOverrideRole(final String... overrideRoles) {
```

Comment:
We no longer allow default of SYSTEM_USER.  If no overrideRoles are passed in, nothing will match